src/components/Organisms/Header/Nav/Nav.js

```diff
@@ -97,28 +89,26 @@ const MainNav = ({ navItems }) => {
                 index={index}
                 isSubMenuOpen={!!isSubMenuOpen[group.id]}
               >
-                {!isMobile ? (
-                  <Text>
-                    <NavLink
-                      href={thisUrl}
-                      inline
-                      rel={relNoopener}
-                      aria-haspopup={hasPopUp}
-                      onKeyUp={keyPressed(group.title)}
-                    >
-                      {thisFirstChild.title}
-                    </NavLink>
-                  </Text>
+                {isMobile ? (
```

Comment:
As I say, I find it hard to parse, but I _think_ larger touch devices with screens wider than `sizes.nav` e.g. tablets won't see the dropdowns

However, guessing the touch detection was removed because it didn't actually work? If so maybe this is something to look at later?

Reply:
For screens larger than `nav` desktop header will show.. don't think we need to worry now for touch screens bigger than 1150px.

Reply:
Was thinking of ipads in landscape but guess it's not gonna be many people

Reply:
yeah don't think so, still.. the open/close is only for the mobile menu (burger menu) .. at this size 1150 desktop header will display.

Reply:
Yeh I totally get that. I just meant that e.g an ipad pro in landscape would show the desktop header despite being a touch device. So users wouldn't be able to get to the dropdowns, as it has no hover state. Ideally whether to employ touch behaviour would be primarily based on whether it's a touch device rather than viewport width. But appreciate it's an edge case and detecting touch would make it more complicated!

Reply:
ah ok ok. yeah, I believe you still can open the dropdown on touch (hard touch) .. but same if I open CR on surface or any laptop with touch screen it will show me the desktop version. Something to be aware in the future!